### ISSUE TITLE
Please list other language bindings of nng

### ISSUE THREAD

**Issue by asuper0** [nanomsg/nng#1613]
**Is your feature request related to a problem? Please describe.**
I'm using more than one coding language. I can't find stable version of nng for them.

**Describe the solution you'd like**
List them in README

**Comment by kant777** [comment]
is there a pure go implementation of this?

**Comment by gdamore** [comment]
Yes.  See mangos - github.com/nanomsg/mangos - it is idiomatic go and supports everything NNG supports except pair1 polyamorous mode (but it can speak as a unicast peer to polyamorous peers) and the ZeroTier transport. 

In fact mangos was the design basis for NNG and has TLS support before any other nanomsg compatible implementation.

I wrote both NNG and mangos.

**Comment by kant777** [comment]
oh I thought mangos is outdated just like nanomsg! so mangos is almost the same as latest and greatest nng?

**Comment by gdamore** [comment]
@kant777 Yes, mangos is *the* thing to use for Go!
